packages/nx/src/utils/package-manager.ts

```diff
@@ -67,6 +67,7 @@ export function getPackageManagerCommand(
       if (gte(pnpmVersion, '6.13.0')) {
```

Comment:
Can you perhaps fix `useExec` to `const useExec = gte(pnpmVersion, '6.13.0');`?
You did it the right way and seeing the `useExec` above kinda hurts the eyes 😅

Reply:
Haha, agreed. Updated.